src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -195,7 +199,8 @@ public void postNotifyUsernameByUsername(String userName, String lang,
           Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
-    logger.debug("postNotifyUseridByUid starting userId='" + userName + "'");
+    logger.debug("postNotifyUsernameByUsername:: parameters: userName = {}, lang = {}, notification = {}, okapiHeaders = {}",
+      userName, lang, Json.encode(notification), okapiHeaders);
```

Comment:
This will generate a code smell during Sonar check. `Json.encode(notification)` needs to be evaluated regardless of the log level. Try to use suppliers.

Reply:
done